src/shared/bg_public.h

```diff
@@ -131,6 +131,8 @@ struct playerState_t
 	int           clips; // clips held
 	int           tauntTimer; // don't allow another taunt until this runs out
 	int           misc[ MAX_MISC ]; // misc data
+
+	bool IsWeaponReady( void ) const;
```

Comment:
Do not put `void` in the parentheses - it is meaningless in C++

Reply:
It's just an (old, dating to my pre-c++ times indeed) habit, which also makes it easier (for me at least) to distinguish between calls and declarations.
I will remove this since that's a problem.